packages/nextjs/pages/index.tsx

```diff
@@ -224,37 +238,142 @@ const Home: NextPage<{ stats: Stats }> = ({ stats }) => {
+                <Image src="/assets/ef-logo.png" alt="EF logo" width={200} height={200} />
+              </div>
+            </div>
+          </div>
+          <div className="flex flex-col items-center">
+            <div className="max-w-md lg:max-w-none">
+              <Image src="/assets/support-high-impact-devs.png" alt="developers list" width={900} height={900} />
+            </div>
+          </div>
+        </div>
+      </div>
+      {/* <div className="bg-base-300">
+        <div className="mx-auto lg:max-w-[1980px] bg-none lg:bg-[url('/assets/support-high-impact-devs.png')] md:[background-position-x:45vw] [background-position-y:50%] md:bg-auto bg-no-repeat ">
           <div className="container max-w-[90%] lg:max-w-6xl m-auto py-16 lg:py-20 lg:px-12 flex flex-col lg:flex-row items-center gap-5 lg:gap-0">
             <div className="text-center lg:text-left sm:w-1/2">
-              <h2 className="text-2xl lg:text-5xl font-semibold my-0 mb-6">
+              <h2 className="text-2xl lg:text-5xl font-semibold my-0 mb-2 lg:mb-6">
                 Supporting
                 <br /> up-and-coming <br />
                 high-impact devs
               </h2>
+              <div className="lg:hidden mb-2">
+                <Image
+                  src="/assets/support-high-impact-devs.png"
+                  alt="Supported list of devs"
+                  width={1115}
+                  height={800}
+                />
+              </div>
+              <p className="lg:w-4/5 m-0 mb-3">
+                Open Developer Streams are a unique way to fund development and give developers at the edges the freedom
+                to build what they think is most important.
+              </p>
+              <p className="lg:w-4/5 m-0 mb-3">
+                Their smart contracts get replenished monthly and allows them to withdraw funds whenever they like by
+                submitting a few sentences about the work or a PR.
+              </p>
+              <p className="lg:w-4/5 m-0 mb-6">
+                This approach produces novel open source solutions and a vibrant learning environment.
+              </p>
+              <p className="lg:mb-3 mt-12 lg:mt-8 text-sm">SUPPORTED BY</p>
+              <div className="flex flex-col gap-6 items-center lg:items-start">
+                <Image src="/assets/ef-logo.png" alt="EF logo" width={200} height={200} />
+              </div>
+            </div>
+          </div>
+        </div>
+      </div> */}
+
+      {/* Cohorts*/}
+      <div className="bg-base-100">
+        <div className="mx-auto lg:max-w-6xl">
+          <div className="container px-4 md:px-12 mx-auto lg:max-w-6xl py-16 lg:py-20 grid lg:grid-cols-[1fr,auto] gap-5 lg:gap-0 items-center">
+            {/* Cohorts Text Content */}
+            <div className="md:w-1/2 lg:w-full md:mx-auto text-center lg:text-left mb-8 lg:mb-0">
+              <h2 className="text-2xl lg:text-5xl font-semibold my-0 mb-6 pr-0 lg:pr-12">
```

Comment:
@technophile-04 should we match the change of Supporting Devs section to have both `h2` with the same size on lg screen size? 

`lg:text-4xl xl:text-5xl`

Reply:
I've just updated it because was adding blank space on small resolutions, and being very close to the other section I think it will look better if both have the same size.

If you prefer another approach, LMK and I'll change it 🙌

Reply:
Ohh yes, thanks for updating it 🙌